wdl2cwl/main.py

```diff
@@ -419,6 +416,31 @@ def get_memory_literal(self, memory_runtime: WDL.Expr.String) -> float:
 
         return memory
 
+    def get_outdir_requirement(
+        self, outdir: Union[WDL.Expr.Get, WDL.Expr.Apply]
+    ) -> int:
+        """Produce the memory requirement for the output directory from WDL runtime disks."""
+        with WDLSourceLine(outdir, ConversionException):
+            if isinstance(outdir, (WDL.Expr.Apply, WDL.Expr.String)):
+                list_object = (
+                    outdir.arguments if hasattr(outdir, "arguments") else outdir.parts  # type: ignore
+                )
+                for obj in list_object:
+                    if isinstance(
+                        obj, (WDL.Expr.Get, WDL.Expr.Apply, WDL.Expr.Placeholder)
+                    ):
+                        return self.get_outdir_requirement(obj)  # type: ignore
+            elif isinstance(outdir, (WDL.Expr.Get, WDL.Expr.Placeholder)):
+                expr = self.get_expr(outdir)
+                expr_str = expr
+                if isinstance(outdir, (WDL.Expr.Placeholder)):
```

Comment:
if it's a placeholder. the tralslate_wdl_placeholder function will return with the string `$(expr)`. This removes the first two and last chars of the string